src/AWS.Deploy.ServerMode.Client/ServerModeSession.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using AWS.Deploy.ServerMode.Client.Exceptions;
+
+namespace AWS.Deploy.ServerMode.Client
+{
+    /// <summary>
+    /// Helper class that allows launching deployment tool in server mode.
+    /// </summary>
+    public class ServerModeSession
+    {
+        private const int TCP_PORT_ERROR = -100;
```

Comment:
We are also declaring this constant here https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.CLI/CommandReturnCodes.cs#L36
Can we unify this an reference 1 place?

Reply:
`AWS.Deploy.ServerMode.Client.csproj` doesn't (can't) reference CLI. It is an independent package which will be used by the VS Toolkit.

Reply:
I am not saying that we should reference the CLI, but we should think about unifying these things maybe by using a shared project. I am sensing a pattern here and I don't want to end up having to maintain 2 places.